md/single-sign-on-with-saml.md

```diff
@@ -201,13 +201,13 @@ For example on linux, you can use the command ssh-keygen, then go to “cd ~/.ss
             </SP>
        </keycloak-saml-adapter>
    ```
-7. If your Identity Provider is corectly configured (see the section *Configure the Identity provider*), you are done.  
+7. If your Identity Provider is corectly configured (see the section *Configure the Identity Provider*), you are done.  
 Then you can try to access a portal page, an app page or a form URL (or just `http://<host>:<port>/bonita[?tenant=<tenantId>]`) and make sure that you are redirected to your Identity Provider to log in (unless you are already logged in).  
 Note that if you try to access `http://<bundle host>:<port>/bonita/login.jsp`, then you won't be redirected as this page still needs to be accessible in order for the tenant administrator (or another user if you set the property `saml.auth.standard.allowed` to true) to be able to log in without an account on the Identity Provider.
 
 ::: info
```

Comment:
Maybe there is no need to introduce the logout here since a whole chapter is dedicated to that later on

Reply:
Good point. I will remove it